### DISCUSSION TITLE
Change default /downloads mount to new location

### DISCUSSION THREAD

**Discussion by RandomNinjaAtk** [RandomNinjaAtk/docker-lidarr-extended#18] [Polls]
To possibly avoid conflicts, adjust the default volume internal mount location to a more unique one.

**Comment by RandomUser345** [comment]
Voted no because I think this container should follow the single responsibility principle. If RandomNinjaAtk needs to adapt it for each and every possible integration this will become a mess and we're back at AMD again.

**Reply by Protektor-Desura** [reply]
Well I just thought it would be cleaner to not download in to the same directory as my torrents for music. To have it's own directory similar to AMD.

**Reply by RandomNinjaAtk** [reply]
I think I am planning to implement the change because if it can reduce support issues, that is probably a better choice...

But I do agree with @RandomUser345 statement about following the single responsibility principle, its impossible to maintain and adapt for every possible edge case. But this may reduce the amount of edge cases...

**Reply by Protektor-Desura** [reply]
I would think you are correct that a different directory for the scripts downloads from the torrent downloads would probably result in fewer support issues but that's just a guess.

**Comment by RandomNinjaAtk** [comment]
This was implemented in the latest update/release.
